crates/miden-lib/asm/miden/active_note.masm

```diff
@@ -86,18 +86,18 @@ export.get_recipient
     # => [RECIPIENT]
```

Comment:
Can you also update inputs to storage in all `RECIPIENT` descriptions (search for `RECIPIENT is the ...`)? For example, for `get_recipient` here:

```
RECIPIENT is the commitment to the active note's script, storage, the serial number.
```